uint8 state
uint8 NAVIGATION_STATE_LONG_RANGE = 0
uint8 NAVIGATION_STATE_SHORT_RANGE = 1
